kolibri/plugins/learn/assets/src/vue/error-page/index.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
We can probably do a general purpose API error handling thing in the core app, in the future.